wrappers/src/app_cs.hpp

```diff
@@ -133,6 +133,22 @@ namespace binding {
     extern std::function<BsonCallbackT> s_bson_callback;
     extern std::function<ApiKeysCallbackT> s_api_keys_callback;
 
+    inline auto get_string_callback_handler(void* tcs_ptr) {
+        return [tcs_ptr](const std::string* response, util::Optional<AppError> err) {
+            if (err) {
+                std::string error_category = err->error_code.message();
+                MarshaledAppError app_error(err->message, error_category, err->link_to_server_logs, err->http_status_code);
+
+                s_bson_callback(tcs_ptr, realm_value_t{}, app_error);
```

Comment:
chose to reuse the `s_bson_callback` as it might be tedious to introduce a `s_string_callback`, but that is definitely an option that is worth discussing here